Integrations/Cloaken/Cloaken.py

```diff
@@ -70,3 +73,39 @@ def get_client():
     else:
         # server error or unavailable
         return_error('Error Cloaken Unshorten: ' + str(response.get('data', 'key missing')))
+
+if demisto.command() == 'cloaken-screenshot-url':
+    client = get_client()
+    url = demisto.args()["url"]
+    screenshot = utility.RasterizeAndRetrieveImage(client, url=url)
+    context={"Url":url, "Status":'failed'}
+    try:
+        result = screenshot.get_screenshot()
+        stored_img = fileResult(result.get("filename","screenshot.png"), result.get("data","NA"), "image")
+        demisto.results({
+            'Type': entryTypes['image'],
+            'ContentsFormat': formats['text'],
+            'File': stored_img['File'],
+            'FileID': stored_img['FileID'],
+            'Contents': ''
+        })
+        context["Status"] = "Success"
+        return_outputs(
+            tableToMarkdown("Screenshot Succeeded", context),
+            {"CloakenScreenshot":context},
+            context
+            )
+    except utility.RasterizeTimeout as e:
+        context["Exception"] = str(e)
+        return_outputs(
+            tableToMarkdown("Screenshot Timed Out", context),
+            {"CloakenScreenshot":context},
+            context
+            )
+    except utility.RasterizeException as e:
+        context["Exception"] = str(e)
+        return_outputs(
```

Comment:
use `return_error` instead.

Reply:
We do not want the integration to error if the rasterization fails.  That would stop the users workflow.  This is why we return a status.